save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -294,4 +294,26 @@ class VulnerabilityController(
     }.map {
         ResponseEntity.ok("Project $projectName was successfully deleted in vulnerability $vulnerabilityName")
     }
+
+    @PostMapping("/save-user")
+    @Operation(
+        method = "POST",
+        summary = "Save new date.",
+        description = "Save new date.",
+    )
+    @PreAuthorize("permitAll()")
+    @ApiResponse(responseCode = "200", description = "Successfully saved new user in vulnerability")
+    @ApiResponse(responseCode = "404", description = "Could not either find user or find vulnerability")
+    @ApiResponse(responseCode = "409", description = "Could not save date as it is already present in vulnerability")
+    fun saveUser(
+        @RequestBody userName: String,
+        @RequestBody vulnerabilityName: String,
+        authentication: Authentication,
+    ): Mono<StringResponse> = blockingToMono {
+        vulnerabilityService.saveUser(userName, vulnerabilityName)
```

Comment:
Do we want to provide roles?

Reply:
I think in the future we will add a role. But now I don't know which is better.

Reply:
In the future it will cause a big refactoring